text/bayes.go

```diff
@@ -160,6 +161,30 @@ type NaiveBayes struct {
 	Output io.Writer
 }
 
+// histogram allows conncurrency-friendly map access via its
+// exported Get and Set methods
+type histogram struct {
```

Comment:
could we call this `concurrentMap` or something else? `histogram` isn't that relevant of a name